src/Symfony/Component/Serializer/Normalizer/NormalizerInterface.php

```diff
@@ -59,7 +59,7 @@ public function supportsNormalization(mixed $data, string $format = null, array
      * Use type "object" to match any classes or interfaces,
      * and type "*" to match any types.
      *
-     * @return array<class-string|'*'|'object'|string, bool|null>
+     * @return array<'*'|'object'|class-string|string, bool|null>
```

Comment:
I'd let "phpstan-like types" first

Reply:
Currently, Fixer does not support such logic.
Feel free to introduce it via [raising PR](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/compare)